crates/oxc_linter/Cargo.toml

```diff
@@ -26,7 +26,7 @@ oxc_cfg = { workspace = true }
 oxc_codegen = { workspace = true }
 oxc_diagnostics = { workspace = true }
 oxc_ecmascript = { workspace = true }
-oxc_index = { workspace = true }
+oxc_index = { workspace = true, features = ["serialize"] }
```

Comment:
@Boshen adding this breaks some builds and I'm not sure why. For example: https://github.com/oxc-project/oxc/actions/runs/11545707859/job/32133003696?pr=6974#step:4:100

Reply:
Update: `just bench` compiles correctly on my machine o.O

Reply:
Found it: `oxc_sytax` needed the `serialize` feature enabled too